.github/workflows/_e2e.yml

```diff
@@ -90,6 +90,8 @@ jobs:
 
       - name: Build explorer
         if: inputs.isTypescriptSDK || inputs.isExplorer || inputs.isRpc || github.ref_name == 'develop'
+        env:
+          IOTA_NETWORKS: '{ "localnet": { "id": "localnet", "name": "Localnet", "url": "http://localhost:9124", "explorer": "http://localhost:3000", "chain": "iota:local", "faucet": "http://localhost:9123" } }'
```

Comment:
What happened before?

Reply:
It was using network configs from `sdk/.env.defaults`, like all the other steps in this workflow still do.
Here we override the config for Explorer to force it to use the indexer endpoint